libs/email-builder/src/lib/components/RequestBrazeSetUpMessage.tsx

```diff
@@ -0,0 +1,61 @@
+import { renderToStaticMarkup } from 'react-dom/server';
+import type { NewsletterData } from '@newsletters-nx/newsletters-data-client';
+import type { MessageContent } from '../types';
+import { MessageFormat } from './MessageFormat';
+import { NewsletterPropertyTable } from './NewsletterPropertyTable';
+
+interface Props {
+	pageLink: string;
+	newsletter: NewsletterData;
+}
+
+export const RequestBrazeSetUpMessage = ({ pageLink, newsletter }: Props) => {
+	return (
+		<MessageFormat
+			title={
+				<>
+					Please create a new Braze campaign for newsletter "
+					{newsletter.identityName}"
+				</>
+			}
+		>
+			<p>
+				The Braze for this newsletter are below - if these need to change for
+				any reason, please contact the newsletters team.
```

Comment:
```suggestion
				any reason, please <a href={pageLink}>update in the newsletters tool</a> .
```

We have exposed these elements for `TagEditors` in the form